src/components/ResourceList/components/FilterControl/components/DateSelector/tests/DateSelector.test.tsx

```diff
@@ -134,9 +134,7 @@ describe('<DateSelector />', () => {
 
       expect(wrapper.find(DatePicker).exists()).toBe(false);
     });
-  });
 
-  describe('filterValue', () => {
```

Comment:
The above describe block is also called "filterValue", so put all the tests together